asistente_ladm_col/asistente_ladm_col_plugin.py

```diff
@@ -1155,6 +1155,13 @@ def show_wiz_col_party_cad(self, *args):
     @db_connection_required
     @survey_model_required
     def show_dlg_group_party(self, *args):
+
+        # TODO: is disabled until the migration is completed.
+        self.logger.warning_msg(__name__,
+            QCoreApplication.translate("AsistenteLADMCOLPlugin",
+                                       "This functionality is being migrated, it will be available again soon."))
+        return
```

Comment:
I think we should do this in another pull request. Something like, disable unsupported functions.

Reply:
I agree, the adjustment is made in commit ab1926c